client/js/screens/SignIn/SignIn.js

```diff
@@ -15,136 +19,192 @@ import InputDefaultField from '../../components/InputDefaultField';
 import InputDefaultCheckBox from '../../components/InputDefaultCheckBox';
 import {THEME} from '../../config';
 import styles from './styles';
+import {addViewer} from '../../config/models';
+import PropTypes from 'prop-types';
 
-const SignIn = props => {
+const client = new ApolloClient({uri: 'http://157.245.163.7:8000/'});
```

Comment:
this should be initialized somewhere else, as I'm sure the Apollo client is used in not just the signin page